src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -2291,19 +2316,24 @@ public abstract partial class DbParameterCollection : System.MarshalByRefObject,
         public abstract object SyncRoot { get; }
         object System.Collections.IList.this[int index] { get { throw null; } set { } }
         object System.Data.IDataParameterCollection.this[string parameterName] { get { throw null; } set { } }
+        int System.Collections.IList.Add(object? value) { throw null; }
```

Comment:
@roji Your PR is adding new public APIs in the ref file. Please make sure to document them with triple slash comments so the reviewers can sign off your change. We want to make sure all PRs introducing new public APIs get properly documented before getting merged.

These are explicit interface implementations (EIIs) so it's ok if you copy-paste the documentation for the original APIs. You can add a remark with a message that [looks like this](https://docs.microsoft.com/en-us/dotnet/api/system.data.dataset.system-componentmodel-ilistsource-getlist?view=netcore-3.1) (replace the two [keywords]):

```
<remark>
This member is an explicit interface member implementation. It can be used only when the [instanceName] instance is cast to an [interfaceName] interface.
</remark>
```

Reply:
@carlossanlop I've revived this and added XML docs following the model for DataSet.IListSource.GetList which you linked to. If you prefer me to copy the actual docs from IList I can do that (or do we may support `<inheritdoc />`?

Reply:
@carlossanlop, rather than requiring such copy/paste, can we just automatically handle that for explicitly implemented members?

Reply:
FYI have removed the new-api-needs-documentation (as the boilerplate XML docs have been copied in).

Reply:
The [DocsPortingTool](https://github.com/carlossanlop/DocsPortingTool) is now able to detect an EII API and automatically add the sentence I suggested if it does not yet exist in triple slash.
Feel free to remove that sentence from this PR. I wrote the original suggestion back when the feature wasn't available in the tool.

Reply:
Great to hear that was added. Thanks.